docs/source/intro.rst

```diff
@@ -11,4 +11,57 @@ As a programmatic interface to Ansible, the SDK lets you:
 * Marshall parameters for operations to different formats.
 * Pass data in native Python structures to and from Ansible.
 
-You can execute Ansible content with the SDK in any combination of local or remote subprocesses with or without containerization.
\ No newline at end of file
+SDK architecture
+----------------
+
+Ansible SDK runs in your Python application and takes the following inputs:
+
+Project
+    Logical collection of Ansible playbooks, credentials, and inventories.
+Credentials
+    Authentication with host systems.
+Inventory
+    List of host systems on which automation jobs run.
+
+With those inputs, Ansible SDK makes multi-threaded, asynchronous calls to perform automation runs against your project payload.
+You can execute Ansible content with the SDK in any combination of local or remote subprocesses.
+You can also use execution environments, which are container images packaged with all dependencies.
+
+Local execution
+***************
+
+Ansible SDK calls the local Ansible executor to perform automation runs.
+The local Ansible executor is the runtime on the application host that pulls execution environments, runs your automation jobs, and returns status to Ansible SDK.
+
+.. image:: images/sdk-local-executor.svg
+    :width: 750
+    :height: 1000
+    :align: center
+    :alt: This diagram illustrates how Ansible SDK takes inputs and calls the local Ansible executor to perform an automation run.
+
+Remote execution
+****************
+
+Ansible SDK calls the Automation mesh controller node to perform automation runs.
+The controller node is the central component that pulls execution environments and runs your automation jobs across remote nodes in the Automation mesh.
+The controller node also returns status to Ansible SDK at the end of each automation run.
+
+.. image:: images/sdk-remote-executor.svg
```

Comment:
so receptor manages the EEs totally on its own and we don't have to do anything?

Reply:
@eqrx I think that's the case, but I've only tried with a cluster that I set up locally. @Akasurde would have a better answer.

In any case your question made me re-think this chunk of text. I've tried to simplify it down a little. What do you think of the change?